es66x/build.gradle

```diff
@@ -53,7 +53,7 @@ compileScala {
 dependencies {
     compile project(path: ':core')
     compile             group: 'org.elasticsearch',                 name: 'elasticsearch' ,             version: esVersion
-//    compileClasspath    group: 'org.codelibs.elasticsearch.lib',    name: 'plugin-classloader',         version: esVersion // no 6.8.11&12 in mvn atm
```

Comment:
I think you cannot uncomment it. That's why pipeline failed. There are no these two version published. This dependency is only needed to run the project with Idea.

Reply:
aah ok, I it didn't work in idea without that and I thought that they are available.